src/rfquack_radio.h

```diff
@@ -29,89 +29,94 @@
 #include "rfquack.pb.h"
 
 #include <cppQueue.h>
+#include <radio/RFQnRF24.h>
+#include <radio/RFQCC1101.h>
+
+#include "defaults/radio.h"
+
+#ifndef RFQUACK_RADIO_RX_QUEUE_LEN
+#define RFQUACK_RADIO_RX_QUEUE_LEN RFQUACK_RADIO_RX_QUEUE_LEN_DEFAULT
+#endif
 
 /*****************************************************************************
  * Variables
  *****************************************************************************/
 Queue rfquack_rx_q(sizeof(rfquack_Packet), RFQUACK_RADIO_RX_QUEUE_LEN, FIFO,
                    true);
 
-/* Radio instance */
-RFQRadio rfquack_rf(RFQUACK_RADIO_PIN_CS, RFQUACK_RADIO_PIN_RST,
-                    RFQUACK_RADIO_PIN_IRQ);
+/*
```

Comment:
Make clear that this is an example and/or do not reference `RFQnRF24` specifically.